app/api/files/jsRoutes.js

```diff
@@ -47,36 +47,66 @@ export default app => {
     multer().any(),
     captchaAuthorization(),
     (req, _res, next) => {
-      req.body = JSON.parse(req.body.entity);
-      return next();
+      req.body.entity = JSON.parse(req.body.entity);
```

Comment:
Already discussed with @konzz:
- wrap this in a try/catch to prevent any uncaught errors due to malformed requests.

Reply:
Done